docs/release-notes/2.33.0.md

```diff
@@ -13,28 +13,28 @@ Percona Monitoring and Management (PMM) is an open source database monitoring, m
 ## Release Highlights
 
 
-### Data loss prevention for PMM server outages
+### Offline metric collection during PMM server outages or loss of PMM client-server network connectivity
 
-Starting with PMM 2.33.0, we have introduced a new solution to prevent data loss in PMM. A network outage between the PMM server and the client prevented metrics from being collected. As a result, the historical data was lost, which can be useful when analyzing trends, diagnosing problems, or investigating issues from a previous period.
+Starting with PMM 2.33.0, we have introduced a new solution to prevent data collection interruption in PMM, via offline metric collection. A network outage between the PMM server and the client prevented metrics from being collected. As a result, the historical data was lost, which can be useful when analyzing trends, diagnosing problems, or investigating issues from a previous period.
 
 In order to address the problem mentioned above, for the PMM server and PMM client running version 2.33.0, in case of a brief outage or network issue, metrics are stored on the monitored instance until the PMM server is up and running. At this point, the metrics will be pushed to the server. It only applies to PMM Clients that run in push mode, which is the default mode. The same holds when a brief network outage occurs on the client side.
 
 !!! caution alert alert-warning "Caution"
     The client node can currently store only 1 GB of offline data. So, for example, if your instance is down for three days, all the data will not be retrieved.
 
-With the implementation of the above feature, we have taken the first step toward PMM HA. Furthermore, we are happy to provide a roadmap for PMM HA implementation, a much-anticipated solution by our users. In the event of a failover, your PMM setup will experience near-zero downtime, and monitoring will continue without any data loss.
+With the implementation of the above feature, we have taken the first step toward PMM HA. Furthermore, we are happy to provide a roadmap for PMM HA implementation, a much-anticipated solution by our users. In the event of a failover, your PMM setup will experience near-zero downtime, and monitoring will continue without any data collection interruption.
```

Comment:
instead of "collection interruption", highlighting the same wording " will continue with the offline metric collection" would be better? what do you think?

Reply:
I agree.